src/EventStore.TestClient/Commands/WriteFloodProcessor.cs

```diff
@@ -121,7 +121,7 @@ internal class WriteFloodProcessor : ICmdProcessor {
 						if (localAll % 100000 == 0) {
 							var elapsed = sw2.Elapsed;
 							sw2.Restart();
-							context.Log.Trace(
+							context.Log.Verbose(
```

Comment:
```suggestion
							context.Log.Debug(
```

If we leave this on verbose, it won't get logged out